docs/containers/bridge-to-kubernetes.md

```diff
@@ -20,141 +20,67 @@ In this guide, you will learn how to use Bridge to Kubernetes to redirect traffi
 * [Visual Studio Code][vs-code] running on macOS, Windows 10, or Linux (currently in preview).
 * The [Bridge to Kubernetes][btk-vs-code] extension installed in Visual Studio Code.
 
-> **Note** Although this quickstart works with Azure Kubernetes Service (AKS), you can also try Bridge to Kubernetes with other Kubernetes clusters. Support for other clusters is in preview.
-
-## Create a Kubernetes cluster
-
-Create an AKS cluster in a [supported region][supported-regions]. The below commands create a resource group called `MyResourceGroup` and an AKS cluster called `MyAKS`.
-
-```azurecli-interactive
-az group create \
-    --name MyResourceGroup \
-        --location eastus
-az aks create \
-    --resource-group MyResourceGroup \
-    --name MyAKS \
-    --location eastus \
-    --node-count 3 \
-    --generate-ssh-keys
-```
-
-## Install the sample application
-
-Install the sample application on your cluster using the provided script. You can run this script on your development computer or using the [Azure Cloud Shell][azure-cloud-shell]. Use the name of your cluster and resource group.
-
-> **Important**: You must have **Owner** or **Contributor** access to your cluster in order to run the script.
-
-```azurecli-interactive
-git clone https://github.com/Microsoft/mindaro
-cd mindaro/
-chmod +x ./bridge-quickstart.sh
-./bridge-quickstart.sh -g MyResourceGroup -n MyAKS
-```
-
-Navigate to the sample application running your cluster by opening its public URL, which is displayed in the output of the installation script.
-
-```console
-$ ./bridge-quickstart.sh -g MyResourceGroup -n MyAKS
-Checking directory /home/<user>/mindaro for GIT repo Microsoft/Mindaro
-Setting the Kube context
-...
-To try out the app, open the url:
-bikeapp.bikesharingweb.EXTERNAL_IP.nip.io
-```
-
-In the above sample, the public URL is `bikeapp.bikesharingweb.EXTERNAL_IP.nip.io`.
-
 ## Connect to your cluster and debug a service
 
-On your development computer, download and configure the Kubernetes CLI to connect to your Kubernetes cluster using [az aks get-credentials][az-aks-get-credentials].
-
-```azurecli
-az aks get-credentials --resource-group MyResourceGroup --name MyAKS
-```
-
-Open `mindaro/samples/BikeSharingApp/Bikes` from the [Bike Sharing sample application][bike-sharing-github] in Visual Studio Code. Open the Azure Kubernetes Service extension and select the **bikeapp** namespace in the **MyAKS** cluster. Right-click the **bikeapp** node, and choose **Use Namespace**.
+On your development computer, make sure your current context is set to the cluster and namespace in which your application is running.
 
-![Select Namespace](images/bridge-to-kubernetes-vs-code/select-namespace.png)
+Open the workspace for the app you want to debug in Visual Studio Code. Open the Command Palette (`kb(workbench.action.showCommands)`), and run the command **Bridge to Kubernetes: Configure** to start the configuration process.
```

Comment:
Please consider replacing `(kb(workbench.action.showCommands))`, that is not very actionable, by `(View > Command Palette...)` that most users will understand better.

Reply:
That is an artifact of the Preview environment - it would resolve to the appropriate command keys for the platform on the user's machine.